_maps/map_catalogue.txt

```diff
@@ -90,10 +90,15 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 47)(y = 47)(z = 1)
 		Tags = "Boss Combat Challenge", "Major Loot", "Shelter", "Lava"
 
-		File Name = "_maps\RandomRuins\BeachRuins\lavaland_crashed_starwalker.dmm"
+		File Name = "_maps\RandomRuins\LavaRuins\lavaland_crashed_starwalker.dmm"
```

Comment:
I'm going to let this slip through because i'm epic and hilarious.

Reply:
I believe most if not all of these concerns have been addressed now, I've updated my PR's images to reflect the new version of the ruin.